thirdparty/jaspr_bootstrap/lib/src/components/table.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:jaspr/jaspr.dart';
+import 'package:jaspr/ui.dart';
+
+import 'base.dart';
+
+enum TableType {
```

Comment:
This should be named as TableVariant I think..: https://getbootstrap.com/docs/5.2/content/tables/#variants